modules/configuring-log-storage-cr.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+// * logging/cluster-logging-loki.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-log-storage-cr_{context}"]
+= Configuring log storage
+
+You can configure which log storage type your {logging} uses by modifying the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo} and an internal log store that is either the LokiStack or Elasticsearch.
++
+include::snippets/logging-elastic-dep-snip.adoc[]
```

Comment:
Note states "will be removed" but it reads to me that this could be in the current release. Maybe state a future release?

Reply:
Can the note fall under the last bullet point as it gets hidden under the large note or add a plus sign to indent the note with the third bullet point?

Reply:
Made some updates to the wording in the note